cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -315,12 +315,16 @@ class CloudInfraCatalogItem(BaseCatalogItem):
 
     @property
     def fill_dict(self):
+        # In 5.10 catalog name is appended with 'My Company'
```

Comment:
I think this would be more readable as a `VersionPicker`, and a bit more versatile as its a `fill_dict` property, that would be used to fill a view, where the version pick will be automatically resolved.

Thoughts?

```
from cfme.utils.version import VersionPicker, LOWEST

cat_name = VersionPicker(
    LOWEST: getattr(self.catalog, 'name', None),
    '5.10': 'My Company/{}'.format(getattr(self.catalog, 'name', None))
)
```

Reply:
This looks like the same implementations and dict returns in both classes, and they both inherit `BaseCatalogItem`. Please double check me on this, and move this method to `BaseCatalogItem` if that's the case.

Reply:
@mshriver I dont think we have to go with `VersionPick`. specifally where we are providing data to view. Yes, `VersionPick` is  best option in `Views` but other than `View` no need to import `VersionPicker` as its dynamic data generation. As per my understanding for `locators` assignment except `ParametrizeView` we con't deciside vesions so `VersionPick` is right but other palces where we can use `appliance.version`.